### PR TITLE
Add "BlogMVC" app.

### PR DESCRIPTION
**Pull request by Kareylo**
https://github.com/Kareylo/BlogMVC-CakePHP3

Part of the BlogMVC Project : https://github.com/Grafikart/BlogMVC
So, there's no tests, it's just to see how CakePHP 3 works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -419,6 +419,7 @@ Additional lists you might find useful:
 - [Sandbox](http://sandbox.dereuromark.de) - A sandbox CakePHP application with lots of demos and plugin showcasings.
 - [Query Examples](https://github.com/lorenzo/cakephp3-examples) Advanced query building examples.
 - [Xeta](https://github.com/XetaIO/Xeta) - A resource to help people starting with CakePHP.
+- [BlogMVC](https://github.com/Kareylo/BlogMVC-CakePHP3) - A simple Blog example with CakePHP based on [BlogMVC Project](https://github.com/Grafikart/BlogMVC)
```
Comment (dereuromark) [inline README.md]: Please respect alphabetical order. Full stop missing at the end.

**Comment by dereuromark** [comment]
I think you should remove the empty files in your app, those are only noise and make it look more complicated :)
You can use my [Setup plugin shell](https://github.com/dereuromark/cakephp-setup) if you want to: `bin/cake setup.clear empty_folders`

**Comment by Kareylo** [comment]
Great idea, i'll do it !

**Comment by jeremyharris** [comment]
There is French text in the Markdown plugin helped, not sure if that's a problem. Also the plugin looks a little incomplete and I'm not sure where the vendor file was copy pasted from. (Does it need a license?) Is suggest using the common mark plugin from the php league as it's a popular php package for converting Markdown.

**Comment by Kareylo** [comment]
All requests done, sorry for the long time to correct all of that mistakes.

**Pull request merged**